modules/virt-editing-vm-web.adoc

```diff
@@ -14,31 +14,40 @@
 ifeval::["{context}" == "virt-editing-vm-template"]
 :virt-vm-template:
 :object: virtual machine template
-:object-gui: Virtual Machines
-:object-vm-overview: VM Template Details
+:object-gui: Templates
+:object-vm-overview: Virtual Machine Overview
```

Comment:
Shouldn't the value of this variable be VM Template Details?

Reply:
I believe the correct values are displaying for each conditional expression to account for the difference between VMs and VM Templates:

ifeval::["{context}" == "virt-editing-vm-template"]
:virt-vm-template:
:object: virtual machine template
:object-gui: Templates
:object-vm-details: VM Template Details
endif::[]

ifeval::["{context}" == "virt-edit-vms"]
:virt-vm:
:object: virtual machine
:object-gui: Virtual Machines
:object-vm-details: Details
endif::[]